src/main/java/uk/gov/hmcts/reform/judicialapi/controller/advice/InvalidRequestException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.judicialapi.controller.advice;
+
+public class InvalidRequestException extends RuntimeException {
+
```

Comment:
create single exception and throw that exception where ever possible. eg. JrdException extend RuntimeExcception. Lets reduce no of exception classes.

Reply:
I have followed the same pattern as we currently have in the other repos, e.g. prd, crd. This is used to differentiate better the different error responses.